README.md

```diff
@@ -19,20 +19,24 @@ Are you interested in seeing how much money a candidate raised? Or spent? How mu
 betaFEC is a collaboration between [18F](http://18f.gsa.gov) and the FEC. It aims to make campaign finance information more accessible (and understandable) to all users.
 
 ## FEC repositories
-We welcome you to explore, make suggestions, and contribute to our code. 
+We welcome you to explore, make suggestions, and contribute to our code.
 
 This repository, [openFEC](https://github.com/18F/openfec), is home to betaFEC’s API.
 
 ### All repositories
 - [FEC](https://github.com/18F/fec): a general discussion forum. We [compile feedback](https://github.com/18F/fec/issues) from betaFEC’s feedback widget here, and this is the best place to submit general feedback.
 - [openFEC](https://github.com/18F/openfec): betaFEC’s API
+- [swagger](https://github.com/18F/swagger-ui):this generates our interactive API documentation
 - [openFEC-web-app](https://github.com/18f/openfec-web-app): the betaFEC web app for exploring campaign finance data
 - [fec-style](https://github.com/18F/fec-style): shared styles and user interface components
 - [fec-cms](https://github.com/18F/fec-cms): the content management system (CMS) for betaFEC
+- [proxy](https://github.com/18F/fec-proxy): this is a light weight app that coordinates the paths between the web app and cms
+- [swagger](https://github.com/18F/swagger-ui):this generates our interactive API documentation
```

Comment:
I meant to move it up, rather than re-add it, since it is closely related to this repo. I'll fix.